README.md

```diff
@@ -1,6 +1,6 @@
 # Jira Plugin for Backstage
 
-![a Jira plugin for Backstage](https://github.com/RoadieHQ/backstage-plugin-jira/docs/jira-plugin.gif)
+![a Jira plugin for Backstage](https://raw.githubusercontent.com/RoadieHQ/backstage-plugin-jira/main/docs/jira-plugin.gif)
```

Comment:
I think this may be more easily done with `docs/jira-plugin.gif` ?  (i.e., a relative ref to an image in the same code repo as this README)

Reply:
I tried that approach but if you publish a package in the NPM registry images fail to loads. It has to be a full path to the image in order to show them on NPM website.